problem_builder/dashboard.py

```diff
@@ -238,10 +238,26 @@ class DashboardBlock(StudioEditableXBlockMixin, XBlock):
         help=_("Toggles if numeric values are displayed"),
         scope=Scope.content
     )
+    header_html = String(
+        display_name=_("Header HTML"),
+        default="",
+        help=_("Additional HTML to include after the heading."),
+        multiline_editor="html",
+        resettable_editor=False,
+        scope=Scope.content,
+    )
+    footer_html = String(
+        display_name=_("Footer HTML"),
+        default="",
+        help=_("Additional HTML to include after the bottom of the page."),
```

Comment:
"After the bottom of the page" was indeed unintentional; this was supposed to read "at the bottom", but I changed it now to be consistent with the header text.